src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  color: #e5e5e5;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
What about last element? Better to reset margins for last elements

Reply:
And it will be more logical to set this star inactive by default, but it's not critical